src/validate.rs

```diff
@@ -733,6 +735,30 @@ fn validate_zulip_group_ids(data: &Data, errors: &mut Vec<String>) {
     });
 }
 
+/// Ensure every member of a team that has a Zulip stream either has a Zulip id
```

Comment:
```suggestion
/// Ensure team members in Zulip streams have a Zulip id
```
I think this reads more clear. If you like it, can you also update the function `validate_zulip_group_ids`?